etc/config/c++.amazon.properties

```
@@ -3858,6 +3858,12 @@ libs.belleviews.description=A library of C++ views that just works for all basic
 libs.belleviews.versions.trunk.version=trunk
 libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/trunk/sources
 
+libs.beman_iterator_interface.name=beman_iterator_interface
```

```suggestion
libs.beman_iterator_interface.name=beman.iterator_interface
```

same here, at least `.name` should be `beman.itereator_interface`.

Check conversation here: 
https://github.com/compiler-explorer/infra/pull/1453#discussion_r1836773092. Maybe the key `libs.beman_iterator_interface.name` can be converted to `libs.beman.iterator_interface.name`, but let's wait the conclusion there.